src/30_Advanced/Video_rotate_to_fullscreen_with_hint.html

```diff
@@ -0,0 +1,127 @@
+<!---{
+"preview": "default",
+"default": "preview",
+"skipValidation": true
+}--->
+<!--
+## Introduction
+**This example is experimental. Feature is not valid AMP yet.**
+
+This is a sample template for implementing rotate-to-fullscreen with a user hint in AMP.
+-->
+<!-- -->
+<!doctype html>
+<html ⚡>
+  <head>
+    <meta charset="utf-8">
+    <title>Live Blog</title>
+    <script async src="https://cdn.ampproject.org/v0.js"></script>
+    <!-- ## Setup -->
+    <!--
+    All scripts for our AMP components must be imported in the header. We will be using three components:
+
+    - `amp-video` to render the video itself.
```

Comment:
It's very useful explaining why you are importing components, could you move the comment above each component?

Reply:
Done.